docker-compose.yaml

```diff
@@ -20,6 +20,15 @@ services:
       - 27017:27017
     command: --bind_ip 0.0.0.0
 
+  # This starts a local PostgreSQL DB.
+  db:
+    container_name: sales_db
+    networks:
+      - shared-network
+    image: postgres:11.1-alpine
+    ports:
+      - 5432:5432
```

Comment:
Make sure this binds on 0.0.0.0 or we will have a problem with compose.

Reply:
Mongo needed that but as far as I can tell postgres doesn't.